public/locales/es/administration.json

```diff
@@ -200,5 +200,15 @@
         "whichSensor": "¿Cuál sensor le gustaría descargar datos?",
         "chooseSensor": "Escoge sensor",
         "anotherSensor": "Descargar datos para otro sensor"
+    },
+    "deleteSensor": {
+        "delete": "Eliminar sensor",
```

Comment:
Borrar is what Hillary had me use for delete on data, so maybe borrar here too? But maybe eliminar makes more sense in the context of a sensor. Not sure.

Reply:
I changed to borrar for consistency with your code--I'll change it back if Hillary recommends doing so